coderd/httpapi/queryparams.go

```diff
@@ -79,6 +79,28 @@ func (p *QueryParamParser) Int(vals url.Values, def int, queryParam string) int
 	return v
 }
 
+// PositiveInt32 function checks if the given value is 32-bit and positive.
+// We can't use `uint32` as the value must be within the range  <0,2147483647>
```

Comment:
<nit>Should we include the reason why this is the required range? Is it for a `serial` field?
If so then perhaps we should name it accordingly.</nit>

Reply:
Thanks for the suggestion! I added more context, but I admit I haven't gone too deep with the analysis.